src/main/resources/static/service-info.json

```diff
@@ -1,85 +1,60 @@
 {
   "id": "uk.ebi.eva.eva-seqcol",
-  "name": "Implementation of the GA4GH Sequence collection retrieval API",
+  "name": "Implementation of the GA4GH Sequence collection retrieval API for the European Variation Archive",
   "type": {
     "group": "org.ga4gh",
-    "artifact": "seqcol",
+    "artifact": "refget-seqcol",
     "version": "1.0.0"
   },
   "organization": {
     "name": "European Variation Archive",
     "url": "https://www.ebi.ac.uk/eva"
   },
   "contactUrl": "mailto:eva-helpdesk@ebi.ac.uk",
-  "documentationUrl": "https://www.ebi.ac.uk/eva/....",
-  "updatedAt": "08-24-2023",
+  "documentationUrl": "https://www.ebi.ac.uk/eva/?API",
+  "updatedAt": "2025-03-27",
   "environment": "dev",
   "version": "1.0.0",
   "seqcol": {
     "schema": {
-      "description": "A collection of sequences, representing biological sequences including nucleotide or amino acid sequences. For example, a sequence collection may represent the set of chromosomes in a reference genome, the set of transcripts in a transcriptome, a collection of reference microbial sequences of a specific gene, or any other set of sequences.",
+      "description": "A collection of biological sequences.",
```

Comment:
Should this be something more specific?

Reply:
Just followed the description from the top level schema